arcgis-ios-sdk-samples/Geometry/Project/ProjectViewController.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class ProjectViewController: UIViewController, AGSGeoViewTouchDelegate {
+    @IBOutlet private weak var mapView: AGSMapView!
+    @IBOutlet private weak var stackView: ProjectStackView!
+    private var graphicsOverlay: AGSGraphicsOverlay!
+    private var map: AGSMap!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        (self.navigationItem.rightBarButtonItem as! SourceCodeBarButtonItem).filenames = ["ProjectViewController", "ProjectStackView"]
+        
+        self.map = AGSMap(basemap: .nationalGeographic())
+        self.mapView.map = self.map
+        self.mapView.touchDelegate = self
+        self.mapView.setViewpointCenter(AGSPoint(x: -1.2e7, y: 5e6, spatialReference: .webMercator()), scale: 4e7)
+    }
+    
+    private func createGraphics(coord: AGSPoint) {
```

Comment:
Wonder if we need three methods for creating and adding graphics overlay and graphics? Even so, could you maybe tweak the names a bit? For instance - creating is not the same as creating and adding graphics/overlay to mapView. 

I also think it's fine if we have to initialize the graphics overlay and add it to the mapView in `viewDidLoad()` and then you could just have one method for creating and adding graphics to the overlay. @philium might have a say in what is typically followed for samples.